physikon/Infobrief_PhysiKon_2024.tex

```diff
@@ -0,0 +1,265 @@
+Wenn Sie kreative Ideen haben, wie Sie die Stunde interaktiver nutzen können, bringen Sie diese gerne mit ein!
+Wir helfen gerne bei der Umsetzung.
+
+Inhaltlich könnte Ihre Präsentation folgende Punkte abdecken:
+\begin{itemize}
+    \item Allgemeine Vorstellung des Unternehmens (Standort, Branche, Größe, Aufgabenfelder)
+    \item Arbeitsfelder für MINT-Fachkräfte
+    \item Einstiegsmöglichkeiten
+    \item Falls vorhanden: Typischer Arbeitsalltag, Gehaltsstruktur
+    \item Gerne: Persönlicher Werdegang, Einblick in Ihre persönliche (MINT-)Tätigkeit beim Unternehmen
+\end{itemize} 
+
+Diese Liste ist nicht vollständig und als Anregung gedacht.
+Sie können Ihren Vortrag natürlich beliebig gestalten.
+
+\subsection*{Zeiteinteilung Online-Vorträge}
+
+\begin{table}[h!]
+    \centering
+\begin{tabular}{|c|l|}
+    \hline
+    \multicolumn{2}{|c|}{Montag, 15.04.2024} \\
+    \hline
+    \rowcolor{gray!10} 13 - 14 Uhr & KPMG \\
+    \rowcolor{gray!30} 14 - 15 Uhr & Elmos \\
+    \rowcolor{gray!10} 15 ­- 16 Uhr & Amprion \\
+    \rowcolor{gray!30} 16 - 17 Uhr & Fraunhofer IMS\\
+    \rowcolor{gray!10} 17 ­- 18 Uhr & \\
+    \hline
+    \multicolumn{2}{|c|}{Dienstag, 16.04.2024} \\
+    \hline
+    \rowcolor{gray!10} 13 - 14 Uhr & Zeiss \\
+    \rowcolor{gray!30} 14 - 15 Uhr & Deichmann \\
+    \rowcolor{gray!10} 15 ­- 16 Uhr & Infineon\\
+    \rowcolor{gray!30} 16 - 17 Uhr & Point 8 \\
+    \rowcolor{gray!10} 17 ­- 18 Uhr &  \\
+    \hline
+    \multicolumn{2}{|c|}{Mittwoch, 17.04.2024} \\
+    \hline
+    \rowcolor{gray!10} 13 - 14 Uhr & NRW.BANK \\
+    \rowcolor{gray!30} 14 - 15 Uhr & Cohausz \& Florack \\
+    \rowcolor{gray!10} 15 ­- 16 Uhr & Comnovo \\
+    \rowcolor{gray!30} 16 - 17 Uhr & \\
+    \rowcolor{gray!10} 17 ­- 18 Uhr & \\
+    \hline
+    \end{tabular}
+
+\end{table}
+
+
+\section*{Messetag am 18. April 2024}
+
+\subsection*{Ausstellernamensschilder}
+
+\textbf{Bitte teilen Sie uns bis zum 01.04.2024 die Anzahl und Namen des Standpersonals mit}, damit wir vorab die passenden Namensschilder gestalten können.
+Falls Sie in Ihrem Team Corona-bedingte Ausfälle fürchten, können Sie auch gerne mehr Namen einreichen, dann machen wir die Namensschilder für die eventuelle Vertretung schon mit fertig.
+
+\subsection*{Adresse}
+
+Technische Universität Dortmund\\
+Emil-Figge-Straße 50\\
+44227 Dortmund
+
+\subsection*{Anfahrt}
+
+Wenn Sie mit der Bahn anreisen, erreichen Sie den Campus der TU vom Dortmunder Hauptbahnhof aus mit der S1 Richtung Solingen.
+Der Veranstaltungsort befindet sich in direkter Nähe zur S-Bahn-Station \enquote{Dortmund Universität}.
+
+Wenn Sie mit dem Auto anreisen, stehen Ihnen Parkmöglichkeiten auf dem Parkplatz unter der Mensabrücke am Vogelpothsweg oder an der Emil-Figge-Straße (Einfahrt 18) zur Verfügung.
+Sollten Sie mit schwerem Gepäck anreisen, können Sie mit dem Auto zum Ausladen bis vor das Gebäude der Emil-Figge-Straße 50 fahren.
```

Comment:
Ist das noch aktuell? ich habe im Kopf, dass das im Vorjahr nicht mehr in Ordnung war?